### DISCUSSION TITLE
AWS CDK Is there a way to manage existing ecs services?

### DISCUSSION THREAD

**Discussion by altonotch** [aws/aws-cdk#30040] [Q&A]
I have a service that was deployed by a different tool. I'm trying to see if switching to cdk will help manage the deployments better. So far I am stuck on a case where the service already exists, I can import it via attributes, but I don't see a way to manage it. 
Am I missing anything?

Is there no other way except for doing this outside of the CDK?

**Comment by justin-masse** [comment]
You cannot modify resources not created by your CDK stack. When you import via ARN/ATTRs/name it generally creates a reduced object. For example when you create a DDB table you get a Table object but when you retrieve via attrs/name/arn you get an ITable which only allows you to run grants/etc but not actually modify the resource itself.

You would have to create new ECS resources within your CDK app.
